.gitignore

```diff
@@ -11,11 +11,19 @@ mfakto
 # copied from source directory
 *.cl
 !src/*.cl
+*.exe
 *.h
 !src/*.h
 *.ini
 !src/*.ini
 
+*.o
+*.obj
+*.opensdf
+*.sdf
+*.suo
+*.vcxproj.user
+
```

Comment:
already included under "build artifacts" and "machine-specific Visual Studio files"